workers/unity/Packages/io.improbable.gdk.testutils/Editor/SpatialDeploymentManager.cs

```diff
@@ -224,7 +224,7 @@ private static int KillProcessUnix(int processId, TimeSpan timeout)
         {
             return RunProcessAndWaitForExit(
                 "kill",
-                $"-TERM {processId}",
+                $"-9 {processId}",
```

Comment:
```suggestion
                $"-KILL {processId}",
```

Prefer named over numbers? Since this is a SIGKILL

Reply:
@jessicafalk What was this for again?

Reply:
this is really too long ago xD think some subprocess didn't get killed correctly?
it's definitely needed

Reply:
A `kill -TERM` sends a SIGTERM which gives the process a chance to shut down, `kill -KILL` sends SIGKILL which immediately terminates the process.

Iirc on MacOS the `kill -TERM` wasn't sufficient and was causing tests to fail.

Reply:
I'm oke with merging the change to `-KILL` if others approve